### PR TITLE
DOCSP-32881: Add more mapped results

### PR DESCRIPTION
**Pull request by casthewiz**
### Context
https://jira.mongodb.org/browse/DOCSP-32881

Adds more mapped results 🥇 💯 

### Pitfalls

`mongocli` has a lot of generic slugs mapped to it - and the first entry in the list doesn't match the entry DOCSP-32881, because its slug would be `''` and would match to every landing page for every product.

Theoretically this might actually be fine, since the rest of the boosts should place the `mongocli` docset pages to the top? 
For the sake of expediency, I've replaced with `command/mongocli` to avoid any collision issues there. 

See:
https://docs-search-transport.docs.staging.corp.mongodb.com/search?q=mongocli

Note: Multiple mongocli versions seem to be marked with `includeInGlobalSearch : true`? within staging?

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ trigger:
     include:
       - 'refs/tags/v*'
       - 'refs/heads/main'
+      - 'DOCSP-32881'
```
Comment (casthewiz) [inline .drone.yml:11]:
```suggestion
```

File: .drone.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ trigger:
     include:
       - 'refs/tags/v*'
       - 'refs/heads/main'
+      - 'refs/heads/DOCSP-32881'
```
Comment (casthewiz) [inline .drone.yml:11]:
```suggestion
```

**Review by seungpark — COMMENTED** [review]
the change LGTM but it also warrants unwanted changes. suggesting we tie in project name with the paths, ie.

```
resultMapping = {
  and: [{
    project: 'manual',
    slug: 'reference/operator/query/and'
  }, {
    project: 'manual',
    slug: 'reference/operator/aggregation/and'
  }] , ...
}
```

**Review by sarahemlin — APPROVED** [review]
Looks great!  Thank you so much for getting this in before the Paris Init meeting!

**Comment by casthewiz** [comment]
@seungpark Agreed in spirit with the overlaps in slugs between projects being a problem - but given that the results for these terms are already pretty bad, I've talked with Sarah and we're going to merge this as an incremental improvement. 

I'll track in a separate ticket that we should look at using either `project` as a specifiable field or hook the mapping into fields in the taxonomy

**Comment by seungpark** [comment]
> @seungpark Agreed in spirit with the overlaps in slugs between projects being a problem - but given that the results for these terms are already pretty bad, I've talked with Sarah and we're going to merge this as an incremental improvement.
> 
> I'll track in a separate ticket that we should look at using either `project` as a specifiable field or hook the mapping into fields in the taxonomy

separate ticket SGTM 👍

**Review by seungpark — APPROVED** [review]

**Pull request merged**